tests/console/yast2_dns_server.pm

```diff
@@ -55,7 +55,8 @@ sub run {
     select_console 'root-console';
 
     # Make sure packages are installed
-    zypper_call('in yast2-dns-server bind SuSEfirewall2', timeout => 180);
+    my $firewall_package = is_sle && sle_version_at_least('15') ? 'firewalld' : 'SuSEfirewall2';
```

Comment:
-1 

Will not work on leap and you can expect Tumbleweed to move to firewalld sooner or later

Reply:
Correct but here we have to work with the current dependencies of the yast module which still relies on SuSEfirewall2. I am sure we would break tumbleweed tests right now with changing it for tumbleweed as well. And of course we will be around for leap 15 to fix it then depending on how the situation will look like for leap at the time.